src/c++/perf_analyzer/command_line_parser.cc

```diff
@@ -373,6 +374,12 @@ CLParser::Usage(const std::string& msg)
              "that the average latency is used to determine stability",
              18)
       << std::endl;
+  std::cerr << FormatMessage(
```

Comment:
Need to add documentation

I'd also like to see a comment in the docs that this mode can cause request_rate mode to not reach the desired rate, especially if num_of_sequences is too small.